docs/local_usage.md

```diff
@@ -0,0 +1,48 @@
+# Running the bridge locally and testing it end to end manually from UI yourself
+
+## Local extension Setup:
+ - The tests setup a ganache server with a preset mnemonic. Add the first key from this mnemonic to your Metamask as an extra account to use for testing. You can find this first key here: ARTEMIS_ETHEREUM_KEY in /polkadot-ethereum/test/docker-compose.yml
+ - Add Localhost:8545 to your Metamask and use it
+ - Setup your polkadot-js extension in your browser and create a test Play account
+
+## Local Server Setup:
+ - Run all test docker containers as per the test README
+ - Make sure you have pulled and setup our substrate-ui fork in a seperate folder to this repo (https://github.com/Snowfork/substrate-ui)
+ - Look at the deployed smart contract addresses in the test build config which should have been generated here: /test/build/address.json
+ - Update the bridge react package in your local substrate-ui fork's config with the deployed ETHApp and ERC20App address (In /substrate-ui/packages/page-ethereum-bridge/src/config.tsx)
```

Comment:
My understanding is that the Javascript app in its current state was purely meant for us to demo functionality (in a scripted manner) in zoom calls, etc. I wouldn't really recommend third-parties use it right now as its relatively tedious to configure, setup, and use.

Its actually much easier to just call this 1-liner 
```
truffle exec scripts/sendEth.js <AMOUNT> <SUB_ACCOUNT_ID>
```